src/lib/components/Strip/Strip.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Strip.css';
+
+const Strip = (props) => {
+  let classArray = [];
+  let style = Object.assign({}, props.style);
+  const {
+    colour, image, bordered, padding, children,
+  } = props;
+
+  if (colour) {
+    classArray = classArray.concat(`p-strip--${props.colour}`);
+  }
+
+  if (image) {
+    classArray = classArray.concat('p-strip--image');
+    style = Object.assign(
+      style, { backgroundImage: `url(${image})` },
+    );
+  }
+
+  if (bordered) {
+    classArray = classArray.concat('is-bordered');
+  }
+
+  if (padding === 'shallow' || padding === 'deep') {
```

Comment:
These causes tight coupling between the framework and this project. If we added a new padding style this would need to be updated. I would recommend doing like you did colour above.